backend/lib/use-cases/orders/orders.ts

```diff
@@ -365,9 +390,12 @@ export class OrdersUseCase {
       }
     }
 
-    const remainingChildCases = provisionalOrder.childCases.filter(
-      (c) => !includedCases.includes(c.caseId),
-    );
+    // Make sure we remove the lead case from the remaining child cases if it exists, so we do not
+    // create another consolidation order containing just the lead case.
+    // note -- rejection doesn't have a leadCase
```

Comment:
If we feel we need this context, maybe we should replace the filter lambdas with named functions.